From 4fa98f0b0b493ebbef616404dbc03ba5fe811997 Mon Sep 17 00:00:00 2001
From: userwithuid <userwithuid@gmail.com>
Date: Sun, 27 May 2018 12:47:15 -0700
Subject: [PATCH] fix logic for setting the curl ca bundle file

we do not want to delete the system default by setting this to NULL...
---
 libtransmission/web.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/libtransmission/web.c b/libtransmission/web.c
index db349760b..997a151b5 100644
--- a/libtransmission/web.c
+++ b/libtransmission/web.c
@@ -191,7 +191,10 @@ createEasy (tr_session * s, struct tr_web * web, struct tr_web_task * task)
 #endif
   if (web->curl_ssl_verify)
     {
-      curl_easy_setopt (e, CURLOPT_CAINFO, web->curl_ca_bundle);
+      if (web->curl_ca_bundle != NULL)
+        {
+          curl_easy_setopt (e, CURLOPT_CAINFO, web->curl_ca_bundle);
+        }
     }
   else
     {
-- 
2.17.0

